python/python/glide/async_commands/core.py

```diff
@@ -1576,6 +1697,68 @@ async def type(self, key: str) -> str:
         """
         return cast(str, await self._execute_command(RequestType.Type, [key]))
 
+    async def xadd(
+        self,
+        key: str,
+        values: List[Tuple[str, str]],
+        options: StreamAddOptions = StreamAddOptions(),
```

Comment:
can you make options optional?

Reply:
I am not sure that necessary, since we are giving it a default value

Reply:
it would be better to set it as optional - we don't need to pass the make_stream value if the user doesn't want to send it.